RELEASE/scripts/autoscend/quests/level_04.ash

```diff
@@ -7,6 +20,43 @@ boolean L4_batCave()
 
 	auto_log_info("In the bat hole!", "blue");
 
+	if (possessEquipment($item[bat wings]) && auto_can_equip($item[bat wings]))
+	{
+		if (!get_property("batWingsBatHoleEntrance").to_boolean() && zone_available($location[The Bat Hole Entrance]))
+		{
+			autoForceEquip($item[bat wings]);
+			auto_log_info("Wearing bat wings to get a free bat wing", "green");
+			handleTracker($item[bat wings], $item[bat wing], "auto_otherstuff");
+			return autoAdv($location[The Bat Hole Entrance]);
+		}
+		else if (!get_property("batWingsGuanoJunction").to_boolean() && zone_available($location[Guano Junction]))
+		{
+			if (!provideGuanoStenchResistance())
```

Comment:
Move stench rest check into line 32. Currently, that guy who has bat wings but not 1 stench res is blocked from doing anything in the bat cave because of line 36's return false